src/main/java/mlib/tool/Datum.java

```diff
@@ -17,11 +17,13 @@
  *    You should have received a copy of the GNU General Public License
  *    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-package mSearch.tool;
+package mlib.tool;
 
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+import mlib.tool.Datum;
+
 @SuppressWarnings("serial")
 public class Datum extends Date {
     private final static SimpleDateFormat dateFormatter1 = new SimpleDateFormat("dd.MM.yyyy");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Make "dateFormatter1" an instance variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS2885)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AModifiersOrderCheck)